code/go/0chain.net/chaincore/node/n2n_send.go

```diff
@@ -518,7 +532,7 @@ func ToN2NReceiveEntityHandler(handler datastore.JSONEntityReqResponderF, option
 				})
 			}
 			// TODO:
-			root, _ := context.WithTimeout(common.GetRootContext(), 5*time.Second)
+			root, _ := context.WithTimeout(common.GetRootContext(), 5*time.Second) //nolint:govet
```

Comment:
The govet is complaining the second returned value (the cancel callback function) is not called, usually we should call it with `defer cancel()` to avoid memory leak, but in this situation, we can not do it as the ctx's lifetime is longer than the function, and it won't cause memory leak issue actually.